parsers/IL.py

```diff
@@ -16,48 +16,24 @@
 from datetime import datetime
 from logging import Logger, getLogger
 from typing import Optional
-
+import base64
 import arrow
-from bs4 import BeautifulSoup
 from requests import Session, get
 
-IEC_URL = "www.iec.co.il"
+NOGA_BASE_URL = "https://www.noga-iso.co.il/"
 IEC_PRODUCTION = (
     "https://www.iec.co.il/_layouts/iec/applicationpages/lackmanagment.aspx"
 )
-IEC_PRICE = "https://www.iec.co.il/homeclients/pages/tariffs.aspx"
+NOGA_ELECTRICITY_CONSUMPTION_API = f"{NOGA_BASE_URL}/Umbraco/Api/Documents/GetElectricalData"
+IEC_PRICE = "https://iecapi.iec.co.il//api/content/he-IL?pageRoute=content/tariffs/contentpages/businesselectricitytariff"
```

Comment:
```suggestion
IEC_PRICE = "https://iecapi.iec.co.il/api/content/he-IL?pageRoute=content/tariffs/contentpages/businesselectricitytariff"
```
Should probably be just one slash here right?

Reply:
Actually this is how they do the request for some reason 🤷🏽‍♀️
I'll check to see if it matters

Reply:
Works both ways. changing for convinience